cmd/archive.go

```diff
@@ -97,5 +98,6 @@ func init() {
 	archiveCmd.Flags().SortFlags = false
 	archiveCmd.Flags().AddFlagSet(optionFlagSet())
 	archiveCmd.Flags().AddFlagSet(runtimeOptionFlagSet(false))
+	archiveCmd.Flags().AddFlagSet(configFlagSet())
```

Comment:
@ppcano We need to refactor `configFlagSet` as it would otherwise introduces CLI flags for `archive` that are nonsensical (`--linger`, `--out`, `--no-usage-report`, `--no-thresholds`).